.github/workflows/test-application.yaml

```diff
@@ -13,22 +13,26 @@ jobs:
         runs-on: ubuntu-latest
 
         env:
-            SYMFONY__DATABASE__PASSWORD: root
+            DATABASE_URL: mysql://root:root@127.0.0.1/sulu_redirect_test?serverVersion=5.7
 
         strategy:
             fail-fast: false
             matrix:
                 include:
-                    - php-version: '5.5'
-                      composer-flags: '--prefer-lowest --prefer-dist --no-interaction'
-                      tools: 'composer:v1'
-                      env:
-                          SYMFONY_DEPRECATIONS_HELPER: disabled
-                    - php-version: '7.4'
-                      composer-flags: '--prefer-dist --no-interaction'
-                      tools: 'composer:v2'
-                      env:
-                          SYMFONY_DEPRECATIONS_HELPER: weak
+                    -   php-version: '7.2'
```

Comment:
There's wrong indentation nearly everywhere in this file. Please adjust that

Reply:
I hope I fixed it now 🙈

Reply:
looks good to me 🙂